featurebyte/schema/observation_table.py

```diff
@@ -20,6 +20,7 @@ class ObservationTableCreate(BaseRequestTableCreate):
 
     sample_rows: Optional[int] = Field(ge=0)
     request_input: ObservationInput
+    entity_ids: List[PydanticObjectId] = Field(default_factory=list)
```

Comment:
Curious do we need to link the entity to the column?

Reply:
hm yea i think that makes sense. let me explore / update!